cinema/models.py

```diff
@@ -39,7 +39,7 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
+    genres = models.ManyToManyField(Genre, related_name="movies")
```

Comment:
Consider adding a `related_name` for the `actors` ManyToManyField in the `Movie` model for consistency and easier reverse querying, similar to the `genres` field.